lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -26,6 +26,37 @@ function dosomething_campaign_form_campaign_node_form_alter(&$form, &$form_state
   $form['#after_build'][] = 'dosomething_campaign_form_campaign_node_form_after_build';
 }
 
+function dosomething_campaign_init() {
```

Comment:
two things here
1. it's typically standard to have `init` hooks at the top of the module file
2. maybe this should live in `dosomething_global`